src/GovUk.Education.ExploreEducationStatistics.Content.Services/ReleaseService.cs

```diff
@@ -100,13 +148,27 @@ public class ReleaseService : IReleaseService
                 );
         }
 
-        public Task<Either<ActionResult, CachedReleaseViewModel?>> GetCachedRelease(
-            string publicationSlug, string? releaseSlug = null)
+        private static void FilterContentBlock(IContentBlockViewModel block)
```

Comment:
OK I see now that it's lifted and shifted from existing code - if this is for the purposes of just stripping comments from the Content, I'd be tempted that we mention that explicitly in either the method name or the parameter `FilterRegex`, as the work being doe wasn't immediately clear.  I'd perhaps call `FilterRegex` `CommentsRegex` instead personally or something similar with the method name, although I appreciate that it then does away from any future abstraction here.

Anyhoo, not for this PR.

Reply:
Yeah I didn't look at this too closely because it's largely been a lift and shift of the existing code from Publisher. It's filtering out comments before the release is cached.

I've used your suggestion to rename `FilterRegex` to `CommentsRegex` now to make it a bit clearer. Thanks.

Reply:
Cool ta